src/stories/Library/search-result-page/FacetLineSelected.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react";
+import { Tag } from "../tag/Tag";
+import { FacetLineItem } from "./FacetLine";
+
+export interface FacetLineSelectedProps {
+  items: FacetLineItem<"term">[];
+}
+
+const FacetLineSelected: FC<FacetLineSelectedProps> = ({ items }) => {
+  return (
+    <ul className="facet-line-selected">
+      {items.map(({ title }) => (
+        <li className="facet-line-selected__item">
```

Comment:
This class is a bit indecisive in what it would like to be? 
.facet-line is the Block
&.facet-line--selected would then need two dashes and is a modifier
&__item is correctly an element

So in this case you would need to apply these classes to the HTML list item: facet-line facet-line--selected and facet-line--selected__item

![image](https://user-images.githubusercontent.com/28546954/199269325-65d0ece2-d708-46a7-b031-17a64e5bd01c.png)
reference: https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/

I would also highly recommend going to front end masters and watching https://frontendmasters.com/courses/sass/bem-introduction/ 
It helped me a lot 😉

Reply:
We went through this. Handled otherwise...